lib/Runtime/Library/JavascriptString.h

```diff
@@ -244,7 +244,9 @@ namespace Js
             static FunctionInfo ToUpperCase;
             static FunctionInfo Trim;
             static FunctionInfo TrimLeft;
+            static FunctionInfo TrimStart;
             static FunctionInfo TrimRight;
```

Comment:
Any reason why we are keeping a lot of the old TrimLeft/TrimRight references around? I feel like we should be able to effectively change the names entirely and then arbitrarily tack on the start/end entrypoints as trimStart/trimEnd in InitializeStringPrototype.

Reply:
As for possibly redundant objects -- we will follow up with a clean-up commit